source/views/settings/screens/credits.tsx

```diff
@@ -27,45 +56,27 @@ const markdownStyles: MarkdownProps['styles'] = {
 	},
 }
 
-const Title = glamorous.text({
-	textAlign: 'center',
-	marginTop: 10,
-	marginBottom: 5,
-	...Platform.select({
-		ios: iOSUIKit.largeTitleEmphasizedObject,
-		android: material.headlineObject,
-	}),
-})
+const Title = (props: TextProps) => (
+	<Text {...props} style={[styles.title, props.style]} />
+)
 
-const Heading = glamorous(Title)({
-	...Platform.select({
-		ios: iOSUIKit.subheadEmphasizedObject,
-		android: material.titleObject,
-	}),
-})
+const Heading = (props: Parameters<typeof Title>[0]) => (
+	<Title {...props} style={[styles.heading, props.style]} />
+)
 
-const About = glamorous.text({
-	...Platform.select({
-		ios: iOSUIKit.bodyObject,
-		android: material.body1Object,
-	}),
-	paddingHorizontal: 25,
-	paddingTop: 10,
-})
+const About = (props: TextProps) => (
+	<Text {...props} style={[styles.about, props.style]} />
+)
 
-const Contributors = glamorous(About)({
-	...Platform.select({
-		ios: iOSUIKit.footnoteEmphasizedObject,
-		android: material.body1Object,
-	}),
-	textAlign: 'center',
-})
+const Contributors = (props: TextProps) => (
+	<Text {...props} style={[styles.contributors, props.style]} />
+)
 
 const formatPeopleList = (arr: Array<string>) =>
 	arr.map((w) => w.replace(' ', ' ')).join(' • ')
```

Comment:
What is going on with the replace call?

Reply:
I believe it replaces spaces with non-breaking spaces, so that people's names don't wrap?